src/paths.ts

```diff
@@ -1,9 +1,9 @@
 const IN = "/in";
-const STUDY = `${IN}/study`;
+const STUDY = `/study`;
 
 const paths = {
   root: "/",
-  home: STUDY,
+  home: `${IN}${STUDY}`,
```

Comment:
Note to self (@eecavanna) to look into this on my own: this path has the prefix, but paths like `studyCreate` don't.